tests/console/python_flake8.pm

```diff
@@ -0,0 +1,36 @@
+# SUSE's openQA tests
+#
+# Copyright (C) 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+# Summary: Regression test for python-flake8
+# Maintainer: Avinesh Kumar <akumar@suse.de>
+
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils 'zypper_call';
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    # make sure that python-flake8 is installed
+    zypper_call 'in python3-flake8';
+
+    # Test case 1: check if flake8 is functional
+    assert_script_run('mkdir /tmp/empty_dir');
+    assert_script_run('flake8 /tmp/empty_dir');
+
+    # Test case 2: check if flake8 is working as expected
+    assert_script_run('curl -O ' . data_url("python/sample.py"));
+    validate_script_output('flake8 --exit-zero sample.py', sub { m/E265 block comment should start with '# '/ && m/F401 'os' imported but unused/ });
+    assert_script_run('rm sample.py');
```

Comment:
```suggestion
    script_run('rm sample.py');
    script_run("rmdir /tmp/empty_dir");
```

Perhaps we also want to remove the `empty_dir`?

Reply:
tbh,it can even stay there... since it's in /tmp and a few kb of a script would not hurt :)

Reply:
rmdit won't work if the dir is not empty anyway

Reply:
@grisu48 , Thank you!

And well, I have updated the commit to remove the empty_dir also.